### ISSUE TITLE
matTooltip doesn't announce contents on NVDA screen reader when user focuses on button with tooltip

### ISSUE THREAD

**Issue by mertdeg2** [angular/material2#11749]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
When the tooltip trigger gets keyboard focus, screen readers also announce the tooltip

#### What is the current behavior?
Tooltip contents are not annonuced

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-material2-issue-fmnuwv?file=app/app.component.html
![unnamed-screen-capture](https://user-images.githubusercontent.com/16231118/41245105-f0e95004-6d5b-11e8-88b7-80207adfe970.gif)

In above GIF, i am pressing TAB to get to button element.

#### What is the use-case or motivation for changing an existing behavior?
Better a11y

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
material2/Windows/NVDA

**Comment by crisbeto** [comment]
A gid probably isn't the best way to illustrate it, but it seems to work for me with NVDA:

![demo](https://user-images.githubusercontent.com/4450522/41309207-67c3f436-6e7e-11e8-9920-b7753db7118f.gif)

**Comment by josephperrott** [comment]
Closing as we have not been able to reproduce it and are not getting any more reports of this issue.
